modules/con_notifications-configuration.adoc

```diff
@@ -98,23 +98,23 @@ spec:
 apiVersion: argoproj.io/v1alpha1
 kind: NotificationsConfiguration
 metadata:
- name: default-notifications-configuration # <1>
+ name: default-notifications-configuration #<1>
 spec:
- subscriptions: |
-  subscriptions: | # <2>
-    # subscription for on-sync-status-unknown trigger notifications
+ subscriptions: #<2>
+  subscriptions: |
+#subscription for on - sync - status - unknown trigger notifications
     - recipients:
-    - slack:test2
-    - email:test@gmail.com
+      - slack:test2
+      - email:test@gmail.com
     triggers:
-    - on-sync-status-unknown
-    # subscription restricted to applications with matching labels only
+      - on-sync-status-unknown
+#subscription restricted to applications with matching labels only
     - recipients:
-    - slack:test3
+      - slack:test3
     selector: test=true
```

Comment:
`selector` and `triggers` should be at same indendation level as `recipients`

Check out the below sample available at https://argo-cd.readthedocs.io/en/stable/user-guide/subscriptions/#default-subscriptions
```
...
...
  subscriptions: |
    # subscription for on-sync-status-unknown trigger notifications
    - recipients:
      - slack:test2
      - email:test@gmail.com
      triggers:
      - on-sync-status-unknown
    # subscription restricted to applications with matching labels only
    - recipients:
      - slack:test3
      selector: test=true
      triggers:
      - on-sync-status-unknown
```

Reply:
Fixed.